### PR TITLE
fix(install): Change fatal error message when failing to install

### PR DESCRIPTION
**Pull request by Julien4218**

### CODE REVIEW

**Review by sanderblue — APPROVED** [review]

File: internal/install/command.go [hunk 1]
```diff
@@ -56,7 +55,7 @@ var Command = &cobra.Command{
 
 			// Run the install.
 			if err := i.Install(); err != nil {
-				log.Fatalf("Could not install New Relic: %s, check the install log for details: %s", err, config.DefaultLogFile)
+				log.Fatalf("We encountered an error with the installation, if this problem persists please visit the documentation and support page for additional help here: https://one.nr/06vjAeZLKjP")
```
Comment (zlesnr) [inline internal/install/command.go:58]: Consider changing "with the installation" to "during installation".  Also here, the "if this" should be a new sentence.

**Review by zlesnr — DISMISSED** [review]

File: internal/install/command.go (hunk 1 above)
Comment (Julien4218) [reply internal/install/command.go:58]: I've copy/pasted what Rishi requested

Comment (zlesnr) [reply internal/install/command.go:58]: As written, it looks like two sentences.  I approved the PR.

Comment (Julien4218) [reply internal/install/command.go]: Yeah, I agree. I've updated.

**Pull request merged**